build/templates/olm-artifacts-template.yaml.tmpl

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: olm-artifacts-template
+
+parameters:
+- name: REGISTRY_IMG
+  required: true
+- name: CHANNEL
+  value: staging
+  required: true
+- name: IMAGE_TAG
+  value: latest
+  required: true
+
+objects:
+- apiVersion: hive.openshift.io/v1alpha1
+  kind: SelectorSyncSet
+  metadata:
+    generation: 1
+    name: configure-alertmanager-operator-catalogsource
```

Comment:
Since it's a single SSS it doesn't make sense to have CatalogSource in the name anymore.
Maybe just `configure-alertmanager-operator` ?

Reply:
WFM